src/components/calcite-button/calcite-button.scss

```diff
@@ -60,6 +60,17 @@
   margin-inline-end: var(--calcite-button-content-margin);
 }
 
+.icon-start-empty {
+  .content {
+    margin-left: unset;
```

Comment:
@anveshmekala can you unset the CSS logical property here? not left/right values.